packages/backend-common/src/reading/tree/TarArchiveResponse.ts

```diff
@@ -105,6 +105,7 @@ export class TarArchiveResponse implements ReadTreeResponse {
       files.push({
         path,
         content: () => content,
+        lastModifiedAt: entry.mtime ? new Date(entry.mtime) : undefined,
```

Comment:
I'm a little cautious of this - Not sure what `mtime` will resolve to and maybe it just makes sense to not implement it here unless we're pretty confident that it is actually modified time in the tar archive and not insertion time, or when it was extracted to disk or any other possible value it could be.

Reply:
Good point. It was a while back now, but I believe when I tested `readTree` on GitHub archives the time seemed to be the commit time of that file. However, I can't say if this is universally true and I should double check as well. Will do some additional checking to see if this is the case or not.

Reply:
So, I tested with GitHub and the `mtime` for every file in the archive seems to be based on the latest commit. So, thinking we should probably just not set the `lastModifedAt` for individual files in TarArchiveResponse's. However, I can't say this would be the case for all usages, but seems likely this would hold true for Git based readers at least.

Reply:
Is the mtime of readTree response items and important bit of this implementation or was it something you added because it made sense? I think that given that the mtime doesn't immediately seem very useful It might be best to hold off on implementing it for readTree responses. If it is important for the S3 response then perhaps we could only have it be present there? Overall I'm feeling that there's a considerable risk that the mtime provided by GitHub is more likely to confuse people than help them

Reply:
@Rugvip - You are right, it's currently only useful to us on the S3 readTree response which doesn't uses `fromReadableArray` rather then `fromTarArchive`. So it would be ok to leave `ReadTreeResponseFile.lastModifiedAt` by have it always undefined for `fromTarArchive` & `fromZipArchive`? That would be fine for us at the moment.